tests/spread/plugins/godeps/legacy-build-run/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Build and run a basic Godeps snap with no base
+
+environment:
+  SNAP_DIR: ../snaps/godeps
+  GOBIN/gobin: gobin
+  GOBIN/no_gobin:
+
+systems: [ubuntu-*]
+
+prepare: |
+  # Make sure no additional snaps are installed before running this test.
+  $TOOLS_DIR/restore.sh
+
+restore: |
+  cd "$SNAP_DIR"
+  snapcraft clean
+  rm -f ./*.snap
+
+execute: |
+  if [ -z "$GOBIN" ]; then
+    unset GOBIN
+  fi
```

Comment:
I'm confused, why have the variants define GOBIN if the test itself unsets it?

Reply:
This is basically a copy from "run"